File: trunk/java/log_client_java/src/com/persiste/sdk/LogserviceResponse.java
package com.persiste.sdk;

import java.util.ArrayList;

public class LogserviceResponse {
	private String status;
	private String message;
	private ArrayList<Object> data;
	
	
	public LogserviceResponse(String status, String message) {
		this.status=status;
		this.message=message;
	}
	
	public void setData(ArrayList<Object> data){
		this.data=data;
	}
	
	public String getStatus(){
		return this.status;
	}
	
	public String getMessage(){
		return this.message;
	}
	
	public ArrayList<Object> getData(){
		return this.data;
	}
		
}

File: csharp/log_client_cs/PersisteDotSdk/LogServiceResponse.cs
/**
* Persiste. C# SDK
*
* The cloud based logging platform.
*
* @package             Persiste. SDK
* @author              Evance Soumaoro
* @copyright           Copyright (c) 2012 - 20xx, Evansofts.
* @license             Evansofts Proprietary Licence (EPL)
* @link                http://evansofts.com
* @version             Version 0.1
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PersisteDotSdk {
    public class LogServiceResponse {
        public String Status { get; set; }
        public String Message { get; set; }
        public List<Object>  Data {get;set;}

        public LogServiceResponse(String status, String message) {
            this.Status=status;
            this.Message = message;
            this.Data = null;
        }

    }
}
